README.md

```diff
@@ -321,44 +321,44 @@ returns a promise rejection.  The only exception to that behaviour is if the
 handler does not provide the `applyMethod` optimization trap.  Then, its
 default implementation is
 ```js
-HandledPromise.applyFunction(HandledPromise.get(p, prop), args)
+Promise.applyFunction(Promise.get(p, prop), args)
 ```
 
 This expansion requires that the promise for the remote method be unnecessarily
 reified.
 
 For an unfulfilled handler, the trap's `target` argument is the unfulfilled
-handled promise, so that it can gain control before the promise is resolved.
+delegated promise, so that it can gain control before the promise is resolved.
 For a fulfilled handler, the method's `target` argument is the result of the
 fulfillment, since it is available.
```

Comment:
This makes it sound like the target might be the promise's fulfillment even when that isn't the presence made by `resolveWithPresence`